x/data/client/testsuite/grpc.go

```diff
@@ -329,3 +574,10 @@ func (s *IntegrationTestSuite) TestQueryResolvers() {
 		})
 	}
 }
+
+func encodeBase64Bytes(decodedHash []byte) string {
+	// encode base64 bytes to base64 string
+	encodedHash := base64.StdEncoding.EncodeToString(decodedHash)
+	// replace all instances of "+" with "%2b"
+	return strings.Replace(encodedHash, "+", "%2b", -1)
```

Comment:
just curious, why do we do the replace?

Reply:
For REST endpoints, we cannot use `+` in the param value so it needs to be encoded.